app/vue/src/server/framework-preset-vue.ts

```diff
@@ -14,6 +14,18 @@ export function webpack(config: Configuration) {
           loader: require.resolve('vue-loader'),
           options: {},
         },
+        {
+          test: /\.tsx?$/,
+          use: [
+            {
+              loader: require.resolve('ts-loader'),
+              options: {
+                transpileOnly: true,
+                appendTsSuffixTo: [/\.vue$/],
```

Comment:
@mrmckeb @graup will this run `ts-loader` on all `*.vue` even if the project is not a TS project? Is that an issue?

Reply:
Good question. I'm not 100% sure, but if you remove `lang="ts"` from a TS SCF it stops working, so I assume ts-loader only gets applied if you specify that. Anyway, ts-loader without type checking is basically a no-op for .js files, so it wouldn't cause any issues.

Reply:
I haven't tested, this was from the PR @graup did for the TS preset - my understanding from the docs is that it should operate as @graup says above.